### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by alejandrofdezmerino**

### CODE REVIEW

File: Assignment2/alejandrofdezmerino-AlejandroFernandez.ttl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@base <http://www.rdfsturtle.com/exercise/class>
+@prefix vocab:<http://www.rdfsturtle.com/exercise/class#>
+@prefix vocabd:<http://www.rdfsturtle.com/exercise/class/dispositivo#>
+@prefix vocabm:<http://www.rdfsturtle.com/exercise/class/dispositivo/medida#>
+@prefix vocabu:<http://www.rdfsturtle.com/exercise/class/dispositivo/user#>
+:Measurement8401 vocabm:hasTemperature"29";
```
Comment (ocorcho) [inline Assignment2/alejandrofdezmerino-AlejandroFernandez.ttl:6]: Parsing problems, no datatypes for RDF literals

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by alejandrofdezmerino** [comment]
New update upload

File: Assignment2/alejandrofdezmerino-AlejandroFernandez.ttl [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@base <http://www.rdfsturtle.com/>
+@prefix  d: <http://www.rdfsturtle.com/class#>
+:Measurement8401 d:hasTemperature "29" ;
```
Comment (ocorcho) [inline Assignment2/alejandrofdezmerino-AlejandroFernandez.ttl:3]: Missing datatypes for literals.

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by alejandrofdezmerino** [comment]
New update

**Pull request merged**